### DISCUSSION TITLE
Rename with editor

### DISCUSSION THREAD

**Discussion by noahfraiture** [jesseduffield/lazygit#3736] [Q&A] [answered]
Hey, here's my small config 
```
os:
  editPreset: 'helix'
``` 
When I try to rename a commit in my editor (or the message of a commit with <c-o> it opens nano. It is a bug, a missing feature or am I doing something wrong ?

**Comment by stefanhaller** [comment, marked as answer]
This setting is only used for when you press `e` in lazygit to edit a file. Telling git which editor to use when you commit has nothing to do with lazygit, you do that by doing `git config --global core.editor helix`.

**Reply by noahfraiture** [reply]
Oh great thank you !
